src/libraries/System.Net.Http/src/ILLink/ILLink.Substitutions.mobile.xml

```diff
@@ -3,5 +3,8 @@
     <type fullname="System.Net.Http.HttpClientHandler">
       <method signature="System.Boolean get_IsNativeHandlerEnabled()" body="stub" value="false" feature="System.Net.Http.UseNativeHttpHandler" featurevalue="false" />
     </type>
+    <type fullname="System.Net.Http.HttpClientHandler">
+      <method signature="System.Boolean get_IsNativeHandlerEnabled()" body="stub" value="true" feature="System.Net.Http.UseNativeHttpHandler" featurevalue="true" />
+    </type>
```

Comment:
```suggestion
    <type fullname="System.Net.Http.HttpClientHandler">
      <method signature="System.Boolean get_IsNativeHandlerEnabled()" body="stub" value="false" feature="System.Net.Http.UseNativeHttpHandler" featurevalue="false" />
      <method signature="System.Boolean get_IsNativeHandlerEnabled()" body="stub" value="true" feature="System.Net.Http.UseNativeHttpHandler" featurevalue="true" />
    </type>
```

Why have separate type elements here for the same type? I think it is easier to read putting them together.

Reply:
When this flows to xamarin-android, we have some `.apk` size regression tests, maybe I should add usage of `HttpClient` there?

Reply:
Agreed. Victim of a copy/paste job ;-)

Reply:
> we have some .apk size regression tests, maybe I should add usage of HttpClient there?

This kind of thing will get caught by Maui regression tests (now that we have them). So it might not be critical to also test it in xamarin-android.

btw - what kind of regression tests exist in xamarin-android? Do you verify all the `.dlls` used by a default app?

Reply:
We have a .NET global tool, `apkdiff`, that can output a file [like this](https://github.com/xamarin/xamarin-android/blob/main/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc). This is `dotnet new android`, I think.

Then we have a test that fails if any file grows by a percentage, or overall `.apk` some percentage.

Reply:
Do you think that would be valuable to also test in `dotnet/maui`?

Reply:
I think if we start doing lots of trimming in dotnet/maui, we'll need it it. We should be able to install `apkdiff` and run similar commands in a test.